Assets/Scripts/AnimationsController.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AnimationsController : MonoBehaviour
```

Comment:
Since you are going to have more animations in the game (potentially) let's rename this class to `CandyAnimationsController`

Reply:
Done